api/ccapi/include/optimizer.h

```diff
@@ -121,7 +121,7 @@ class Optimizer {
  * @brief Factory creator with constructor for optimizer
  */
 std::unique_ptr<Optimizer>
-createOptimizer(OptimizerType type,
```

Comment:
and I guess as enum remains, this will also remain??

Reply:
Yes it is added again in #740(To make PR merge faster) There are some untidy dependencies between PRs. I'll look on to the merge order.